README.md

```diff
@@ -57,11 +57,11 @@ Currently supported features:
   * closures
   * bound methods
   * complex numbers (except for arithmetic)
+  * garbage collection (`mark & sweep`, `simple` or `none`)
```

Comment:
1. I don't know whether we should list `simple` and `none` as garbage collectors, as they are not. Also, at the moment the GC is only implemented on ARM Cortex-M.
2. The rest of the code still calls the `simple` GC the `dumb` gc - this should be consistent.

Reply:
Okay, if you want we can probably close this until we have GC for all supported platforms (I thought we had a generic implementation when I opened this).